docs/community_snippets.md

```diff
@@ -344,6 +345,23 @@ Add the Send-MailMessage cmdlet with the most common parameters in a hashtable f
 }
 ```
 
+### New Azure Resource Group
+
+Create a Resource Group on Azure, by @vmsilvamolina.
+
+#### Snippet
+
+```json
+"New Azure Resource Group": {
+    "prefix": "rg",
```

Comment:
If we follow the example set by the `Send-MailMessage` snippet (and other snippets that ship in the extension) perhaps the prefix should be `ex-New-AzureRmResourceGroup`?

Reply:
> If we follow the example set by the `Send-MailMessage` snippet (and other snippets that ship in the extension) perhaps the prefix should be `ex-New-AzureRmResourceGroup`?

@rkeithhill Yes! You´re correct... I modified the snippet name with your suggestion. Thanks for the comment.